### PR TITLE
Chunking deep-dive description

### PR DESCRIPTION
**Pull request by mib-kd743naq**
@alanshaw I still need to add "bibliography", do not merge just yet

### CODE REVIEW

**Comment by mib-kd743naq** [comment]
@alanshaw this is ready to go, thanks for prodding me!
/cc @warpfork @vmx

File: DEEP_DIVES/46-better-file-chunking.md [hunk 1]
```diff
@@ -1,15 +1,75 @@
...
+UX / game-theory-ish problems.**
 
 ## Goal
 
-<Define what we would get if the Open Problem(s) are solved for this Deep Dive>
+It would be great if towards the end of this session we have answers or maybe
+even consensus 🤞 regarding the following deceptively simple
+checklist:
+
+- [ ] Identify a ~dozen-or-two of realistic user-stories where IPFS could
```
Comment (alanshaw) [inline DEEP_DIVES/46-better-file-chunking.md:35]: "~dozen-or-two" - what does this mean?

**Review by alanshaw — APPROVED** [review]
This is a great description! Thank you ❤️

File: DEEP_DIVES/46-better-file-chunking.md (hunk 1 above)
Comment (mib-kd743naq) [reply DEEP_DIVES/46-better-file-chunking.md:35]: @alanshaw I got carried away without thinking D:
Let's add https://github.com/ipfs/camp/pull/59 on top

**Pull request merged**